src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -462,16 +462,55 @@ a {
     color: red;
 }
 
-.MessageComposer {
-    border-top: 1px solid rgba(245, 245, 245, 0.90);
+.MessageComposer_replyPreview .Timeline_message {
+    margin: 0;
+    margin-top: 5px;
+    max-height: 30vh;
+    overflow: auto;
+}
+
+.MessageComposer_replyPreview {
+    background: rgba(245, 245, 245, 0.90);
+    margin: 0px 10px 10px 10px;
+    box-shadow: 0px 0px 5px #91919169;
+    border-radius: 5px;
+}
+
+.MessageComposer_input, .MessageComposer_replyPreview {
     padding: 8px 16px;
 }
 
-.MessageComposer > :not(:first-child) {
+.MessageComposer_replyPreview > .replying {
+    display: inline-flex;
+    flex-direction: row;
+    align-items: center;
+    font-weight: bold;
+}
+
+.MessageComposer_replyPreview > button.cancel {
+    width: 32px;
+    height: 32px;
+    display: block;
+    border: none;
+    text-indent: 200%;
+    white-space: nowrap;
+    overflow: hidden;
+    background-color: transparent;
+    background-image: url('icons/clear.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 18px;
```

Comment:
can has `cursor: pointer` here?

Reply:
Indeed; I think the send and attach file buttons are missing this as well.